feat: Change php exampels and guide towards meta package (#768)
getsentry/sentry-docs: src/collections/_documentation/platforms/php/index.md

## Transport

Sentry PHP is not tied to any specific library that sends HTTP messages. Instead,
it uses [Httplug](https://github.com/php-http/httplug) to let users choose whichever PSR-7 implementation and HTTP client
they want to use.

> Our metapackage that we you should use `sentry/sdk` provides our recommend HTTP client.

Reviewer: ```suggestion
We recommend that you use the `sentry/sdk` meta-package which provides our recommend HTTP client.
```

If you want to use a different HTTP client just install `sentry/sentry` with the clients you prefer:

```bash
composer require sentry/sentry:{% sdk_version sentry.php %} php-http/curl-client guzzlehttp/psr7
```

This will install the library itself along with an HTTP client adapter that uses

> cURL as transport method (provided by [Httplug](https://github.com/php-http/httplug) and a PSR-7 implementation

Reviewer: ```suggestion
cURL as the transport method (provided by [Httplug](https://github.com/php-http/httplug) and a PSR-7 implementation
```
